app/models/concerns/client_search.rb

```diff
@@ -56,7 +56,7 @@ def self.text_searcher(text)
         return none
       end
 
-      client_ids << text if numeric && self.destination.where(id: text).exists? # rubocop:disable Style/RedundantSelf
+      client_ids << text if numeric && GrdaWarehouse::Hud::Client.destination.where(id: text).exists?
```

Comment:
Made this change to always search by  GrdaWarehouse client destination ID, instead of calling `self` (which was throwing an error because hmis client doesn't have a destination scope). We need this anyway to support searching by "Warehouse ID" in the HMIS